vars/mavenCI.groovy

```diff
@@ -61,8 +61,16 @@ def call(body) {
             }
         }
 
+        def testCmd = config.integrationTestCmd
+        if(config.runTestsInUserNamespace == true && testCmd) {
+            def nameSpace = utils.getUsersNamespace()
+            echo "Running the integration tests in the namespace: ${nameSpace}"
+            testCmd = testCmd + " -Dnamespace.use.existing=" + nameSpace
```

Comment:
can't this be handled in the  `mavenIntegrationTest` instead? 

```groovy
mavenIntegrationTest {
  namespace = "foobar"
}
```

If user sets a testCmd, wouldn't they expect to run exactly that?

Reply:
If we only talk about using this library in OSIO Build/Jenkins - then yes. Otherwise - why should I need that?